temp/server-client.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env ruby
```

Comment:
`temp` is a weird place to put this. If it's something we're going to use, I've been leaning towards a `libexec` directory.

Reply:
Makes sense -- it's just an example client for proving the concept, I think if it lives anywhere it'll be in docs

Reply:
I'd rather it in something like `examples/` then.

Reply:
![](https://media.giphy.com/media/l41m35Z4QtUJgVEdy/giphy.gif)